crates/iceberg-catalog/src/api/management/mod.rs

```diff
@@ -834,6 +837,71 @@ pub mod v1 {
         .await
     }
 
+    #[derive(Debug, Deserialize, Serialize, utoipa::IntoParams)]
+    pub struct GetWarehouseStatisticsQuery {
+        /// Next page token
+        #[serde(skip_serializing_if = "PageToken::skip_serialize")]
+        #[param(value_type=String)]
+        pub page_token: PageToken,
+        /// Signals an upper bound of the number of results that a client will receive.
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[serde(default)]
+        pub page_size: Option<i64>,
+    }
+
+    impl GetWarehouseStatisticsQuery {
+        fn to_pagination_query(&self) -> PaginationQuery {
+            PaginationQuery {
+                page_token: self.page_token.clone(),
+                page_size: self.page_size,
+            }
+        }
+    }
+
+    /// Get warehouse statistics
+    ///
+    /// Get statistics about the warehouse, currently table and view counts over time.
+    ///
+    /// We lazily create a new statistics entry every hour, in between hours, the existing entry is
+    /// being updated. If there's a change at created_at + 1 hour, a new entry is created.
+    /// If there's been no change, no new entry is created, meaning there may be gaps.
+    ///
+    /// Example:
```

Comment:
Lets add the information that the examples below are excluding the date information which is present in the message

Reply:
mention that dates are omitted